src/Displaytemplatestandard.h

```diff
@@ -117,55 +111,24 @@ void printScreen()
         else {
             // Show uptime of machine
             u8g2.print(langstring_uptime);
-            float seconds = millis() / 1000;
-            int hours = seconds / 3600;
-            int minutes = (seconds - hours * 60 * 60) / 60;
-
-            // Print hours and minutes
-            u8g2.print(hours);
-            u8g2.print("h ");
-            u8g2.print(minutes);
-            u8g2.print("m");
+            const char* format = "%02lu:%02lu:%02lu";
```

Comment:
Not sure this is a good format. This would look more like a time, not a duration to me, e.g. 00:01:20. I'd prefer the previous format

Reply:
Reverted to the old format